### PR TITLE
Fixed unit playing crashing voice when crashing after being attacked by locomotor warhead

### PR DESCRIPTION
**Pull request by NetsuNegi**
This is a vanilla bug, when a unit begin to crashing after attacked by locomotor warhead, it will play crashing voice and crashing sound. Now it has been fixed, won't play that again.

### CODE REVIEW

File: CREDITS.md [hunk 1]
```diff
@@ -185,7 +185,9 @@ This page lists all the individual contributions to the project by their author.
    - Warhead superweapon launch logic
    - "Shield is broken" trigger event
    - Forbidding parallel AI queues by type
-- **NetsuNegi** - Forbidding parallel AI queues by type
+- **NetsuNegi**
+   - Forbidding parallel AI queues by type
+   - Fixed unit will play crash voice when crashing after attacked by locomotor warhead
```
Comment (Starkku) [inline CREDITS.md:190]: Needs spelling fixes. F.ex

`Fixed unit playing crashing voice when crashing after being attacked by locomotor warhead`

File: docs/Fixed-or-Improved-Logics.md [hunk 2]
```diff
@@ -73,6 +73,7 @@ This page describes all ingame logics that are fixed or improved in Phobos witho
 - Fixed an issue that caused vehicles killed by damage dealt by a known house but without a known source TechnoType (f.ex animation warhead damage) to not be recorded as killed correctly and thus not spring map trigger events etc.
 - `IsAnimated`, `AnimationRate` and `AnimationProbability` now work on TerrainTypes without `SpawnsTiberium` set to true.
 - Fixed transports recursively put into each other not having a correct killer set after second transport when being killed by something.
+- Fixed unit will play crash voice when crashing after attacked by locomotor warhead.
```
Comment (Starkku) [inline docs/Fixed-or-Improved-Logics.md:76]: Same as previous comment.

File: src/Misc/Hooks.BugFixes.cpp [hunk 3]
```diff
@@ -426,6 +426,18 @@ DEFINE_HOOK(0x54D138, JumpjetLocomotionClass_Movement_AI_SpeedModifiers, 0x6)
 	return 0;
 }
 
+DEFINE_HOOK(0x4DACF3, FootClass_CrashingVoice, 0x6)
```
Comment (Starkku) [inline src/Misc/Hooks.BugFixes.cpp:429]: Change hook name to `FootClass_AI_CrashingVoice` to match the convention used elsewhere in Phobos codebase (`ClassName_FunctionName_Application/Purpose`).

**Review by Starkku — CHANGES_REQUESTED** [review]
Code looks fine, some minor nitpicks that should be addressed though.

**Comment by NetsuNegi** [comment]
> Code looks fine, some minor nitpicks that should be addressed though.

Finished

**Review by Metadorius — APPROVED** [review]
Looks good to me, update and make sure all checks pass and then it can be merged.

**Review by chaserli — CHANGES_REQUESTED** [review]
I don't know, I actually don't even consider it a bug

**Pull request closed without merging**